src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -224,6 +224,12 @@ def validate_load_balancer_idle_timeout(namespace):
             raise CLIError("--load-balancer-idle-timeout must be in the range [4,100]")
 
 
+def validate_network_policy(namespace):
+    """validate network policy to be in lowercase"""
+    if namespace.network_policy.islower() is False:
```

Comment:
The CI failed due to this might be None.
```suggestion
    if namespace.network_policy is not None and namespace.network_policy.islower() is False:
```